index.md

```diff
@@ -2,16 +2,86 @@
 layout: lesson
 root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
+date: '`r format(Sys.time(), "%d %B, %Y")`'
```

Comment:
I believe this can go now? We don't use the date variable as the page is not rendered with Rmarkdown.

Reply:
@ledm do you agree?